go/cmd/vtctldclient/command/shards.go

```diff
@@ -80,11 +80,11 @@ that shard.`,
 	// SetShardTabletControl makes a SetShardTabletControl gRPC call to a vtctld.
 	SetShardTabletControl = &cobra.Command{
 		Use:   "SetShardTabletControl [--cells=c1,c2...] [--denied-tables=t1,t2,...] [--remove] [--disable-query-service[=0|false]] <keyspace/shard> <tablet_type>",
-		Short: "Sets the TabletControl record for a shard and tablet type. Only use this for an emergency fix or after a finished MoveTables. The MigrateServedFrom and MigrateServedType commands set this record appropriately already.",
+		Short: "Sets the TabletControl record for a shard and tablet type. Only use this for an emergency fix or after a finished MoveTables.",
 		Long: `Sets the TabletControl record for a shard and tablet type.
 
-Only use this for an emergency fix or after a finished MoveTables. The MigrateServedFrom
-and MigrateServedType commands set this record appropriately already.
+Only use this for an emergency fix or after a finished MoveTables. The MigrateServedType
+commands set this record appropriately already.
```

Comment:
this should be singular "command" now

Reply:
Actually, this command is now gone so I'll remove the entire sentence. Thanks!